OEPS/OEP-5.mediawiki

```diff
@@ -164,3 +197,4 @@ The event must be triggered on any successful calls to approve.
 ===Implementation===
 
 OEP-5 Template: [[https://github.com/ONT-Avocados/python-template/blob/master/OEP5Sample/OEP5Sample_compiler2.0.py | Python Template]]
```

Comment:
The template need to be updated with new interfaces.

Reply:
I can add https://github.com/Ryucoin/RyuNFTs-ONT which implements everything but `approvalForAll` and `tokenMetadata`.

Reply:
ok

Reply:
@Honglei-Cong I added the Ryu contract to the examples